crates/router/src/compatibility/stripe/payment_intents/types.rs

```diff
@@ -490,15 +509,78 @@ impl From<payments::PaymentListResponse> for StripePaymentIntentListResponse {
     }
 }
 
-#[derive(PartialEq, Eq, Deserialize, Clone)]
+#[derive(PartialEq, Eq, Deserialize, Clone, Debug)]
 #[serde(rename_all = "snake_case")]
 pub enum StripePaymentMethodOptions {
     Card {
         request_three_d_secure: Option<Request3DS>,
+        mandate_options: Option<MandateOption>,
     },
 }
 
-#[derive(Default, Eq, PartialEq, Serialize, Deserialize, Clone)]
+#[derive(PartialEq, Eq, Deserialize, Clone, Default, Debug)]
+#[serde(rename_all = "snake_case")]
```

Comment:
since the fields are already in snake_case this is not required I guess.

Reply:
done